packages/SketchMorph2-Core.package/M2Backend.class/instance/undoBlock.st

```diff
@@ -0,0 +1,7 @@
+tool selection
+undoBlock
```

Comment:
Should we name a method based on the return type?

Reply:
Maybe it could be called `undoActions` or anything without using the word `block`